hackertracker/HTInitViewController.swift

```diff
@@ -80,8 +80,8 @@ class HTInitViewController: UIViewController {
         let context = delegate.managedObjectContext!
         
         let df = DateFormatter()
-        df.dateFormat = "yyyy-MM-dd HH:mm:ss"
-        df.timeZone = TimeZone(identifier: "PDT")
+        df.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSXXXXX"
```

Comment:
Creating a date formatter is a fairly expensive operation. Can you add that to the `DateFormatterUtility` so that it is cached?